src/modules/dashboard/components/Dialogs/EscrowFundsDialog/useEscrowFundsDialog.tsx

```diff
@@ -0,0 +1,211 @@
+import { ROOT_DOMAIN_ID } from '@colony/colony-js';
+import React, { useCallback, ReactNode, useMemo, useState } from 'react';
+import classNames from 'classnames';
+import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
+import { COLONY_TOTAL_BALANCE_DOMAIN_ID } from '~constants';
+import ColorTag, { Color } from '~core/ColorTag';
+import { FormSection, SelectOption } from '~core/Fields';
+import { useColonyFromNameQuery } from '~data/generated';
+import { getTokenDecimalsWithFallback } from '~utils/tokens';
+import styles from './EscrowFundsDialog.css';
+import {
```

Comment:
please, move these 3 imports up - in the section with `core` etc (I am providing details of our imports structure in a different comment)